compiler/rustc_parse/src/parser/expr.rs

```diff
@@ -2100,49 +2124,78 @@ impl<'a> Parser<'a> {
         recovered
     }
 
+    /// Keep this in sync with `Token::can_begin_literal_maybe_minus` and
+    /// `Lit::from_token` (excluding unary negation).
+    pub fn eat_token_lit(&mut self) -> Option<token::Lit> {
```

Comment:
```suggestion
    fn eat_token_lit(&mut self) -> Option<token::Lit> {
```
It's only used in this module.